src/main/java/mezz/jei/util/CommandUtilServer.java

```diff
@@ -113,7 +113,7 @@ public static void setHotbarSlot(ServerPlayerEntity sender, ItemStack itemStack,
 			}
 			ItemStack itemStackCopy = itemStack.copy();
 			sender.inventory.setInventorySlotContents(hotbarSlot, itemStack);
-			sender.world.playSound(null, sender.posX, sender.posY, sender.posZ, SoundEvents.ENTITY_ITEM_PICKUP, SoundCategory.PLAYERS, 0.2F, ((sender.getRNG().nextFloat() - sender.getRNG().nextFloat()) * 0.7F + 1.0F) * 2.0F);
+			sender.world.playSound(null, sender.serverPosX, sender.serverPosY, sender.serverPosZ, SoundEvents.ENTITY_ITEM_PICKUP, SoundCategory.PLAYERS, 0.2F, ((sender.getRNG().nextFloat() - sender.getRNG().nextFloat()) * 0.7F + 1.0F) * 2.0F);
```

Comment:
Not sure if `serverPosX` etc work properly here as I have not personally looked at what they do, so if they work fine like this they can probably be left as is, but if there is a slight difference in how they behave you may want to have this (and the other spots that use them), use the same call as before.
The getters for `posX`, `posY`, and `posZ` are as follows:
```
Entity#posX -> Entity#func_226277_ct_()
Entity#posY -> Entity#func_226278_cu_()
Entity#posZ -> Entity#func_226281_cx_()
```

Reply:
Changed these to be on the safe side as serverPos isn't used elsewhere.